charts/cloudzero-agent/docs/sizing-guide.md

```diff
@@ -0,0 +1,104 @@
+### Prometheus Agent Deployment Sizing Guide
+
+#### Introduction
+
+This guide outlines the recommended settings for deploying the Prometheus agent using the CloudZero Helm chart. It includes instructions on how to configure memory and CPU resources based on the size of your cluster.
+
+#### Baseline Memory Requirements
+
+- **Base Memory:** 512Mi
+- **Base Memory Limit:** 1024Mi
+- **Additional Memory:** 0.75Gi per 100 nodes in the cluster
+
+#### Sizing Calculation
+
+It is recommended to consider the shape and size of your prometheus cluster when setting resource memory limits for the prometheus agent. To calculate the memory requirements for your cluster, one can use the following formula:
+
+![sizing formula](./assets/sizing-formula.png)
+
+> This guide uses a simple formula based on number of nodes, however your milage may vary if you have:
+> 
+> * Very large machines, with a large number of pods
+> 
+> * High churn pods or jobs. Each pod started will trigger allocation of a time seriese based metrics cache in the agent. If it restarts, a new cache is created. The cache will resize in memory for the look back period to handle remote write failures (2 hours).
```

Comment:
`seriese` -> `series`

Reply:
Refactored the wording based on feedback from FAM also - dropped the mention of time series :-)

Reply:
oop yeah, sorry, looks like I reviewed mid-change